### DISCUSSION TITLE
Green-Bot Dashboard?

### DISCUSSION THREAD

**Discussion by ghost** [GreenBotDevelopment/Green-bot#38] [Q&A] [answered]
Why is the Green-Bot Dashboard (https://green-bot.app/) not open source?

**Comment by pauldb09** [comment]
For the moment, the dashboard isn't open source but maybe on the future :)

**Reply by ghost** [reply]
Ok that would be really nice

**Comment by ghost** [comment, marked as answer]
Could you make it soon open source because i would like to make pull requests then :)

**Comment by JoniXTech** [comment]
When will be the dashboard public?

I love your Bot ❤️
